src/js/libgeo/GeoOps.js

```diff
@@ -709,6 +801,20 @@ geoOps.CircleMr.updatePosition = function(el) {
     el.matrix = General.withUsage(matrix, "Circle");
     el.radius = r;
 };
+geoOps.CircleMr.getRandomMove = function(el) {
+    // radius 
+    var r;
+    var oldr = el.radius;
+    do {
+        r = CSNumber.add(oldr, CSNumber.real(0.1 * (Math.random() - 0.5)));
+    } while (r.value.real <= 0); // make sure r > 0;
```

Comment:
How about scaling the radius by a random factor instead, to avoid both a change in sign and a complete change of magnitude? Might break for circles of radius zero, though.

Do we even have a problem with circles of negative radius? I think there *should* be no problems, but I'm not sure. Do we allow for complex radii eventually?